src/components/manage/Toolbar/Toolbar.jsx

```diff
@@ -194,9 +194,10 @@ class Toolbar extends Component {
    * @returns {undefined}
    */
   componentDidMount() {
-    this.props.listActions(getBaseUrl(this.props.pathname));
-    this.props.getTypes(getBaseUrl(this.props.pathname));
-    this.props.setExpandedToolbar(this.state.expanded);
+    this.props
+      .listActions(getBaseUrl(this.props.pathname))
+      .then(() => this.props.getTypes(getBaseUrl(this.props.pathname)));
```

Comment:
Can you elaborate more on this than "fix @types missing"?  Again, I'm sure it's obvious to you but I at least could use some edification.  How is `getTypes` related to `listActions`?  I tried reading the definition of the `getTypes` function but neither the documentation comments nor the code says much about what or why it is.

Also again, is this related to my previous changes or cleanup you found along the way?